### PR TITLE
logcat: improve command description

### PR DESCRIPTION
**Pull request by JCarlosR**
Start with Spanish translations for Android commands.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
Welcome to tldr, The translation LGTM seems like you have modified the command description. Can you update the [main page](https://github.com/tldr-pages/tldr/blob/main/pages/android/logcat.md) too? Or can you update the current one to align with other pages?

**Comment by JCarlosR** [comment]
@kbdharun I updated the description in the main page (I think it's better since the previous one is too general). 

I see existing translations in other languages, but I understand the idea is to progressively update them to align with the main page.

**Comment by kbdharun** [comment]
> @kbdharun I updated the description on the main page (I think it's better since the previous one is too general).
> 
> I see existing translations in other languages, but I understand the idea is to progressively update them to align with the main page.

Agreed, Translations can always be updated later, our aim is to simplify and improve the main page which ultimately results in other pages being improved too.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/android/logcat.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # logcat
 
-> Dump a log of system messages.
+> Dump a log of system messages, including stack traces when the device throws an error and messages that you have written from your app with the Log class.
```
Comment (navarroaxel) [inline pages/android/logcat.md:3]: `device` sounds like hardware errors. `written with Log class` is too specific.
```suggestion
> Dump a log of system messages, including stack traces when an error occurred and information messages logged by applications.
```

File: pages.es/android/logcat.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# logcat
+
+> Vuelca un registro de mensajes del sistema, incluidos los seguimientos de pila, los casos de error del sistema y los mensajes que escribes desde tu app con la clase Log.
+> Más información: <https://developer.android.com/studio/command-line/logcat>.
+
+- Mostrar registros del sistema:
```
Comment (navarroaxel) [inline pages.es/android/logcat.md:6]: For Romance languages, the present indicative fits better.
```suggestion
- Muestra registros del sistema:
```

File: pages.es/android/logcat.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`logcat`
+
+- Escribir registros del sistema a un archivo:
```
Comment (navarroaxel) [inline pages.es/android/logcat.md:10]:
```suggestion
- Escribe registros del sistema a un archivo:
```

File: pages.es/android/logcat.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`logcat -f {{ruta/al/archivo}}`
+
+- Mostrar líneas que coincidan con una expresión regular:
```
Comment (navarroaxel) [inline pages.es/android/logcat.md:14]:
```suggestion
- Muestra líneas que coincidan con una expresión regular:
```

File: pages.es/android/logcat.md (hunk 2 above)
Comment (JCarlosR) [reply pages.es/android/logcat.md]: Thanks. I just confirmed we are  doing the same for other Spanish translations.

Comment (navarroaxel) [reply pages.es/android/logcat.md]: Yeah, I try to keep them in sync.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**